CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Change log
 
+## 0.2.1
+
+* #86: Add `hasSelfLoop` into the API.

```

Comment:
This should go to `0.2`, which still hasn't been released.

Reply:
Oops